core/templates/pages/about-page/about-page.module.ts

```diff
@@ -54,10 +57,13 @@ import { ObjectsDomainConstants } from
     SharedComponentsModule
   ],
   declarations: [
-    ServiceBootstrapComponent
+    ServiceBootstrapComponent,
+    AboutPageComponent,
+    TranslatePipe
```

Comment:
Since the translatePipe would be used in multiple modules, shouldn't we add this in the shared module?

(Each pipe and component can be added to only one declaration)

Reply:
Will change that too.

Reply:
Done!